src/columnstore/execution/columnstore_insert.cpp

```diff
@@ -1,106 +1,116 @@
-#include "columnstore/columnstore.hpp"
-#include "columnstore/columnstore_table.hpp"
-#include "duckdb/planner/operator/logical_insert.hpp"
+#include "columnstore/execution/columnstore_insert.hpp"
+#include "duckdb/common/helper.hpp"
 
 namespace duckdb {
 
+//===--------------------------------------------------------------------===//
+// Sink
+//===--------------------------------------------------------------------===//
 class ColumnstoreInsertGlobalState : public GlobalSinkState {
 public:
     ColumnstoreInsertGlobalState(ClientContext &context, const vector<LogicalType> &types,
                                  const vector<unique_ptr<Expression>> &bound_defaults)
-        : executor(context, bound_defaults), insert_count(0) {
+        : executor(context, bound_defaults), insert_count(0), return_collection(context, types) {
         chunk.Initialize(Allocator::Get(context), types);
     }
 
     DataChunk chunk;
     ExpressionExecutor executor;
     idx_t insert_count;
+    ColumnDataCollection return_collection;
 };
 
-class ColumnstoreInsert : public PhysicalOperator {
-public:
-    ColumnstoreInsert(vector<LogicalType> types, idx_t estimated_cardinality, ColumnstoreTable &table,
-                      physical_index_vector_t<idx_t> column_index_map, vector<unique_ptr<Expression>> bound_defaults)
-        : PhysicalOperator(PhysicalOperatorType::EXTENSION, std::move(types), estimated_cardinality), table(table),
-          column_index_map(std::move(column_index_map)), bound_defaults(std::move(bound_defaults)) {}
+SinkCombineResultType ColumnstoreInsert::Combine(ExecutionContext &context, OperatorSinkCombineInput &input) const {
+    return SinkCombineResultType::FINISHED;
+}
```

Comment:
Are you sure we need this? Parallel execution is not supported for this physical operator yet